docs-website/markdown-link-check-config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "ignorePatterns": [
+    {
+    "pattern": "^http://demo\\.datahubproject\\.io"
+    },
+    {
+    "pattern": "^http://localhost"
+    },
+    {
+    "pattern": "^http://www.famfamfam.com"
+    },
+    {
+    "pattern": "^http://www.linkedin.com"
+    },
+    {
+    "pattern": "\\.md$"
+    },
+    {
+    "pattern": "\\.pdl$"
```

Comment:
interesting - most of our pdl and markdown links should work?

what's the reason they're ignored?

Reply:
There are (at least) dozens of deprecated links in the docs because the path in GitHub no longer exists & The docs itself is outdated. I have updated it as much as possible, but i still don’t know the full context in some parts. I think this issue should be addressed in another PR, possibly with the help of the team, rather than just fixing the links. What do you think?